spec/invertedSpec.js

```diff
@@ -0,0 +1,345 @@
+    });
+
+    it('treats documents with the same name and content as one', () => {
+      expect(newIndex.getTitles(testFile)).toEqual(['Alice in Wonderland', 'The Lord of the Rings: The Fellowship of the Ring.']);
+    });
+
+    it('should append " - Copy" to title if title already exists but has different text', () => {
+      expect(newIndex.getTitles(testFile3)).toEqual(['Alice in Wonderland', 'The Lord of the Rings: The Fellowship of the Ring.', 'The Lord of the Rings: The Fellowship of the Ring. - Copy']);
+    });
+
+    it('should include title in array if it has the same text as another but different title name', () => {
+      expect(newIndex.getTitles(testFile2)).toEqual(['Alice in Wonderland', 'The Lord of the Rings: The Fellowship of the Ring.', 'The Lord of the Rings.']);
+    });
+  });
+
+  describe('Tests for the createIndex method', () => {
+    it('should return error message for non-Array objects', () => {
+      expect(() => { newIndex.createIndex('hello', 'Hello World'); })
+          .toThrow(new Error('Invalid file format. Only array of objects can be contained in file.'));
+      expect(() => { newIndex.createIndex('test', new Set(['a', 'the'])); })
+          .toThrow(new Error('Invalid file format. Only array of objects can be contained in file.'));
+      expect(() => { newIndex.createIndex('test', 200); })
+          .toThrow(new Error('Invalid file format. Only array of objects can be contained in file.'));
+    });
+
+    it('should return error message for files with improper key values', () => {
+      const test = [
+        {
+          tite: 'Alice in Wonderland',
+          text: 'Alice. falls- into@ a+ rabbit hole.',
+        },
+      ];
+
+      expect(() => { newIndex.createIndex('doc1.json', invalidTestFile); })
+        .toThrow(new Error('Your JSON file does not have the expected format. Documents are to have title and text keys alone.'));
+      expect(() => { newIndex.createIndex('test.json', test); })
+        .toThrow(new Error('Your JSON file does not have the expected format. Documents are to have title and text keys alone.'));
+    });
+
+    it('should return the correct indices for words in documents', () => {
+      expect(newIndex.createIndex('doc.json', shortFile2)[0]).toEqual({ a: [0],
+        alice: [0],
+        alliance: [1],
+        an: [1],
+        elf: [1],
+        falls: [0],
+        hole: [0],
+        into: [0],
+        man: [1],
+        of: [1],
+        rabbit: [0],
+        unusual: [1] });
+    });
+
+    it('returns an array of file indices and the file name', () => {
+      expect(newIndex.createIndex('doc.json', shortFile2)).toEqual([{ a: [0],
+        alice: [0],
+        alliance: [1],
+        an: [1],
+        elf: [1],
+        falls: [0],
+        hole: [0],
+        into: [0],
+        man: [1],
+        of: [1],
+        rabbit: [0],
+        unusual: [1] }, 'doc.json']);
+    });
```

Comment:
You should consider having the expected result in a variable like this 
` const result = [{ a: [0],
        alice: [0],
        alliance: [1],
         an: [1],
         elf: [1],
         falls: [0],
         hole: [0],
         into: [0],
         man: [1],
     of: [1],
    rabbit: [0],
    unusual: [1] }, 'doc.json']; `
then pass the variable like this 
`.toEqual(result)`. 
This makes your code look clean and readable.